TicketBooking/Controllers/BookingController.cs

```diff
@@ -0,0 +1,67 @@
+using Microsoft.AspNetCore.Mvc;
+using TicketBooking.Helpers;
+using TicketBooking.Models;
+
+namespace TicketBooking.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class BookingController : ControllerBase
+    {
+        private readonly ILogger<BookingController> _logger;
+        private readonly ISeatLockProvider seatLockProvider;
+
+        public BookingController(ILogger<BookingController> logger, ISeatLockProvider seatLockProvider)
+        {
+            _logger = logger;
+            this.seatLockProvider = seatLockProvider;
+        }
+
+        [HttpGet]
```

Comment:
It should be HttpPost because BookingRequest is going to insert/update data in the system